tests/helics/shared_library/FilterTests.cpp

```diff
@@ -36,16 +36,16 @@ TEST_P(filter_type_tests, registration)
     auto mFed = GetFederateAt(1);
 
     helicsFederateRegisterGlobalEndpoint(mFed, "port1", "", &err);
-    helicsFederateRegisterGlobalEndpoint(mFed, "port2", NULL, &err);
+    helicsFederateRegisterGlobalEndpoint(mFed, "port2", nullptr, &err);
     EXPECT_EQ(err.error_code, helics_ok);
     CE(auto f1 = helicsFederateRegisterFilter(fFed, helics_filter_type_custom, "filter1", &err));
     CE(helicsFilterAddSourceTarget(f1, "port1", &err));
-    EXPECT_TRUE(f1 != NULL);
+    EXPECT_TRUE(f1 != nullptr);
```

Comment:
Interesting that it didn't suggest `EXPECT_NE` for these.

Reply:
I suspect it doesn't trigger off pointers, possibly only built-in types like ints,  it also doesn't suggest it for things like fed_id which is a simple wrapper around an integer.